plugins/woocommerce-blocks/tests/e2e/tests/product-gallery/product-gallery.block_theme.side_effects.spec.ts

```diff
@@ -8,6 +8,7 @@ import { Locator, Page } from '@playwright/test';
  * Internal dependencies
  */
 import { ProductGalleryPage } from './product-gallery.page';
+import { initial } from 'lodash';
```

Comment:
This is not used anywhere, so most likely it's some leftover.

```suggestion
```

Reply:
Nice catch! I fixed in 367d06b4242480108c265c13f76b752a27c19991